README.md

```diff
@@ -181,6 +182,12 @@ nats:
   # hostport: "" # nats://{domain or ip}:{port}, if not empty, NATS output is enabled
   # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
 
+stan:
+  # hostport: "" # nats://{domain or ip}:{port}, if not empty, NATS output is enabled
```

Comment:
```suggestion
  # hostport: "" # nats://{domain or ip}:{port}, if not empty, STAN output is enabled
```

Reply:
thanks for the catches, i was blind on those